### ISSUE TITLE
Third state for auto-invites

### ISSUE THREAD

**Issue by seanfarley** [matrix-discord/mx-puppet-discord#132]
I've set autoinvites to nil since I only want to bridge a few channels but I found out that I miss all the direct messages people send me. Would it be possible to have a new state that is "don't invite me to channels but do invite me to direct messages"?

**Comment by ottersarecool** [comment]
I can't find a setting related to disabling autoinvite, where is it?
Also +1 would be a very nice addition

**Comment by seanfarley** [comment]
Yeah, I missed the autoinvite setting for a while. It's `setautoinvite <puppetID> 0` (to disable).

**Comment by Smeat** [comment]
I've been using `bridgechannel <puppetID> <Discord channel ID>` instead of `bridgeguild` if I only want to bridge a few channels.
You can get the ID from the URL: `https://discord.com/channels/<server ID>/<channel ID>`
This requires a bit of manual work for every channel you want to join but seemed like the best solution for my use case.
Maybe this will help your use case as well.

**Comment by seanfarley** [comment]
> I've been using `bridgechannel <puppetID> <Discord channel ID>` instead of `bridgeguild` if I only want to bridge a few channels.

As I mentioned in the originanl post, that's what I do but it causes new direct messages to be completely missed.

> You can get the ID from the URL: `https://discord.com/channels/<server ID>/<channel ID>`
> This requires a bit of manual work for every channel you want to join but seemed like the best solution for my use case.
> Maybe this will help your use case as well.

There's an easier way if you [enable developer mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-). After it's enabled, you can right-click on any channel / user and copy the id.

**Comment by Smeat** [comment]
> As I mentioned in the originanl post, that's what I do but it causes new direct messages to be completely missed.

Sorry, since you didn't mention `bridgechannel` I assumed you might not know about this command.
I've been only using `bridgechannel` without ever using the `bridgeguild` or `setautoinvite` commands. 
This way I won't get any unwanted invites to channels I never directly bridged and direct messages are working just as expected.
But it seems I missunderstood your problem, so just ignore me ;)

> There's an easier way if you [enable developer mode](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-). After it's enabled, you can right-click on any channel / user and copy the id.

Good to know, thank you

**Comment by seanfarley** [comment]
No worries at all :-)
